src/services/logger.ts

```diff
@@ -18,49 +18,127 @@ if (config.nodeEnv !== 'dev' && config.nodeEnv !== 'test') {
   });
 }
 
-const logger = config.nodeEnv === 'dev' || config.nodeEnv === 'test' ? console : log4js.getLogger();
+const devEnv = config.nodeEnv === 'dev' || config.nodeEnv === 'test';
+const logger = devEnv ? console : log4js.getLogger();
+
+const createTimeStamp = (): string => {
```

Comment:
what's the purpose of creating a new method that does exactly what the old method did? And then calling this method from the old method?

Reply:
I shall revert this, I recall there was concern about this function in particular causing flamegraphs to go crazy. I do not remember where I was going with it.